src/nxdoc/tutorials/transversal-how-tos/how-to-implement-local-groups-or-roles-using-computed-groups.md

```diff
@@ -146,6 +146,21 @@ In this how to, you will learn how to let managers of a workspace determine who
 
 {{/callout}}
 
+Starting from 8.10 HF-04 you need to add the following contribution.
```

Comment:
What about: _"Starting from 8.10 HF-04, using virtual groups have to be configured explicitly. You need to add the following contribution to allow it:"_